src/Breakdown/LabelBreakdownScene.tsx

```diff
@@ -69,17 +72,32 @@ export class LabelBreakdownScene extends SceneObjectBase<LabelBreakdownSceneStat
     });
   }
 
+  public getResponsiveSelector(): ResponsiveGroupBySelector {
+    const { responsiveSelector } = this.state;
+    if (!responsiveSelector) {
+      throw new Error('ResponsiveGroupBySelector not initialized');
+    }
+    return responsiveSelector;
+  }
+
   public static readonly Component = ({ model }: SceneComponentProps<LabelBreakdownScene>) => {
     const styles = useStyles2(getStyles);
     const { body } = model.useState();
     const groupByVariable = model.getVariable();
+    const useResponsive = true;
```

Comment:
quick switch to test back and forth. should i remove the old component now or as a follow-up PR